tests/python/unittest/test_meta_schedule_trace_apply.py

```diff
@@ -25,6 +25,9 @@
 from tvm.target import Target
```

Comment:
Please remove change from this file, since VNNI vs AVX512 difference doesn't matter for this unittest.

Reply:
done